apps/src/applab/commands.js

```diff
@@ -59,8 +59,8 @@ function outputError(errorString) {
  */
 function getAsyncErrorHandler() {
   const line = 1 + window.Applab.JSInterpreter.getNearestUserCodeLine();
-  return errorString => {
-    errorHandler.outputError(errorString, ErrorLevel.WARNING, line);
+  return error => {
+    errorHandler.outputError(String(error), ErrorLevel.WARNING, line);
```

Comment:
Fine as-is, but since your block is a single statement you don't really need the block.